hadoop-ozone/dist/src/main/smoketest/snapshot/fs-snapshot.robot

```diff
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I appreciate you adding a smoke test.  I should have done it in my PR (but was too lazy).

Since "no good deed goes unpunished", I'm going to ask you to do a bit more.

Please do an "fs -ls" on "$snapshot1"  and on "$snapshot1/key1" and confirm that they work correctly as well.